openshift/templates/issuer-web/config/openvp_rev/dev/config.json

```diff
@@ -14,21 +14,21 @@
     "oidcSettings": {
       "authority": "https://toip-vc-authn-controller-dev.apps.silver.devops.gov.bc.ca",
       "clientId": "openvp-dev",
-      "redirectUri": "https://openvp-issuer-dev.apps.silver.devops.gov.bc.ca/oidc-callback",
-      "redirect_uri": "https://openvp-issuer-dev.apps.silver.devops.gov.bc.ca/oidc-callback-error",
+      "redirectUri": "https://openvp-candy-issuer-dev.apps.silver.devops.gov.bc.ca/oidc-callback",
```

Comment:
This needs to be reverted, as it is for an existing profile. Applies to other files in `../config/openvp_rev` as well.

I think you meant to update ` openshift/templates/issuer-web/config/openvp-candy_rev/dev/config.json` instead.

Reply:
Argh thanks for the keen eyes